src/scripting/lua_globals.cc

```diff
@@ -384,6 +384,8 @@ static int L_npgettext(lua_State* L) {
       :type script: :class:`string`
       :arg script: The filename relative to the root of the data directory.
       :returns: table returned by included script or an empty table
+         .. versionchanged:: 1.2
+            It returned nil before
```

Comment:
Indentation of this line is wrong. (see in https://github.com/widelands/widelands/actions/runs/6782878008/job/18436005629?pr=6210#step:4:194)

How should I do it correctly?

Reply:
Try to add an empty line before `.. versionchanged`. Probably also one tab less.
Suggestion for the sentence above:

`:returns: The table returned by the included script, or an empty table if no return value is specified.`